### PR TITLE
automatic onboarding emails

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

File: templates/emails/onboarding.day14.noExpenses.hbs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Subject: It's all about transparency
+
+{{> header}}
+
+<h1>Hey {{collective.name}} 👋</h1>
+
+<p>I noticed that you haven't filed any expense yet.</p>
```
Comment (asood123) [inline templates/emails/onboarding.day14.noExpenses.hbs]: either "any expenses" or "an expense"

File: templates/emails/onboarding.day14.noExpenses.hbs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Subject: It's all about transparency
+
```
Comment (piamancini) [inline templates/emails/onboarding.day14.noExpenses.hbs]: add a ! after transparency

File: templates/emails/onboarding.day14.noExpenses.hbs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+<p>At Open Collective we believe that collaborating transparently is the key for a project to succeed. Be open about your costs. Don’t wait until you have a budget in place, go ahead and <a href="https://opencollective.com/{{collective.slug}}/expenses/new">submit expenses</a> to let potential backers know what they're helping fund!</p>
+
+<p>Let us know if you need any help setting things up, we are here to help your collective succeed.</p>
+
+<p>Feel free to schedule a call (<a href="http://calendly.com/piamancini/call">calendly.com/piamancini/call</a>), email me or <a href="https://slack.opencollective.com">join us on our Slack</a>. We would love to know your story.</p>
+
+<p>Best,</p>
+
+<p>Pia, and the Open Collective Team</p>
```
Comment (piamancini) [inline templates/emails/onboarding.day14.noExpenses.hbs]: P**í**a

File: templates/emails/onboarding.day2.hbs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+{{> header}}
+
+<h1>Hey {{collective.name}} 👋</h1>
+
+<p>I’m Pía, one of the founders of Open Collective. I’d love to help you set up and get your collective up and running. Feel free to reach out to me directly replying to this email. I'm here to help you!</p>
+
+<p>Our main piece of advice is this: get your first contributors now! Reach out to your friends & family for them to give you that first boost, even if it’s just for a couple of months. Having those first backers makes a big difference, don’t be shy!</p>
+
+<p>Here is a helpful guide: <a href="https://medium.com/open-collective/ten-steps-to-successful-open-source-crowdfunding-fa2b43e82687">Ten Steps to Successful Open Source Crowdfunding</a></p>
```
Comment (piamancini) [inline templates/emails/onboarding.day2.hbs]: This  link was a button on the emails I was sending. Is it one here? I think it makes sense.

File: templates/emails/onboarding.day2.hbs [hunk 5]
```diff
@@ -0,0 +1,21 @@
+Subject: Give visibility to your backers and sponsors
```
Comment (piamancini) [inline templates/emails/onboarding.day2.hbs]: This is the subject of day 7. Subject for day two is "Welcome! - some quick advice from our team."

File: templates/emails/onboarding.day7.widgets.hbs [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+{{> header}}
+
+<h1>Hey {{collective.name}} 👋</h1>
+
+<p>Did you know that we offer widgets to automatically show your backers and sponsors on your website? It's a great way to thank them for their financial contribution. It will also incentivize others to contribute as well!</p>
+
+<p>Go to the settings of your collective in the "export" tab: <a href="https://opencollective.com/{{collective.slug}}/edit#export">https://opencollective.com/{{collective.slug}}/edit#export</a></p>
+
+<p>And here are some examples: <a href="http://apex.run/#links">Apex</a>, <a href="https://www.spinacms.com">SpinaCMS</a>, <a href="https://webpack.js.org">Webpack</a> (using our API). You can also add a Donate Button or a widget to automatically show your events!</p>
+
```
Comment (piamancini) [inline templates/emails/onboarding.day7.widgets.hbs]: We are not using images here?

File: templates/emails/onboarding.day14.noExpenses.hbs (hunk 3 above)
Comment (xdamman) [reply templates/emails/onboarding.day14.noExpenses.hbs]: ok

File: templates/emails/onboarding.day14.noExpenses.hbs (hunk 2 above)
Comment (xdamman) [reply templates/emails/onboarding.day14.noExpenses.hbs]: ok

File: templates/emails/onboarding.day2.hbs (hunk 5 above)
Comment (xdamman) [reply templates/emails/onboarding.day2.hbs]: good catch

File: templates/emails/onboarding.day2.hbs (hunk 4 above)
Comment (xdamman) [reply templates/emails/onboarding.day2.hbs]: It's quite a very long label for a button.
Also, shouldn't we send this only to open source collectives?

File: templates/emails/onboarding.day7.widgets.hbs (hunk 6 above)
Comment (xdamman) [reply templates/emails/onboarding.day7.widgets.hbs]: Not yet. Cuiki will work on some and we will update then.

File: templates/emails/onboarding.day2.hbs (hunk 4 above)
Comment (piamancini) [reply templates/emails/onboarding.day2.hbs]: I had the button: Get Started! (with the link). 
yes, why do you think it's related?

File: templates/emails/onboarding.day7.widgets.hbs (hunk 6 above)
Comment (piamancini) [reply templates/emails/onboarding.day7.widgets.hbs]: ok

File: templates/emails/onboarding.day2.hbs (hunk 4 above)
Comment (xdamman) [reply templates/emails/onboarding.day2.hbs]: Oh I thought you wanted the link to the 10 steps for successful open source crowdfunding as a button.

So what should get started link to? Simply to the collective?

Like this?

Comment (piamancini) [reply templates/emails/onboarding.day2.hbs]: I had the Get started linked to the 10 steps: 
screen shot 2018-01-17 at 5 31 20 pm
It's up to you really, I think buttons are good CTAs. 2cents. don't block on this.

**Comment by xdamman** [comment]
Ok merging and deploying. @piamancini you are off the hook to manually send those emails :-)
Thanks for having taking the time to do this before this PR was ready!

**Pull request merged**